.gitlab-ci.yml

```diff
@@ -482,33 +482,34 @@ publish:production:s3:scripts:install-mender-sh:
     refs:
       - production
 
-.template:publish:s3:mender-monitor:
+.template:publish:s3:
   stage: publish
   image: debian:buster
   dependencies:
-    # Note that we are using the packages from Debian buster
     - build:packages:debian:buster:amd64
     - build:packages:debian:buster:armhf
     - build:packages:debian:buster:arm64
+    - build:packages:debian:bullseye:amd64
+    - build:packages:debian:bullseye:armhf
+    - build:packages:debian:bullseye:arm64
+    - build:packages:ubuntu:bionic:amd64
+    - build:packages:ubuntu:bionic:armhf
+    - build:packages:ubuntu:bionic:arm64
+    - build:packages:ubuntu:focal:amd64
+    - build:packages:ubuntu:focal:armhf
+    - build:packages:ubuntu:focal:arm64
```

Comment:
do we need to change this: [[arch_indep]="true"](https://github.com/mendersoftware/mender-dist-packages/blob/master/docker-build-package#L42) as well?

Reply:
Why? I don't get it.

It is true that for `mender-monitor` all the `:arm*` jobs won't have valuable artifacts, but it is harmless to fetch them. And this way we are able to use the same template to all commercial packages and avoid duplicating code.

Reply:
> Why?

the only why was: because we are building monitor debs only for amd64.

> 
> It is true that for `mender-monitor` all the `:arm*` jobs won't have valuable artifacts, but it is harmless to fetch them. And this way we are able to use the same template to all commercial packages and avoid duplicating code.

makes sense. I also mis-read, and thoughr we are now building monitor per arch as well. sorry for the noise.